addon/components/admin-input.js

```diff
@@ -1,18 +1,21 @@
 import Ember from 'ember';
 
-var set = Ember.set;
-var get = Ember.get;
-var computed = Ember.computed;
+const set = Ember.set;
+const get = Ember.get;
+const computed = Ember.computed;
```

Comment:
@rwjblue do you know why @clairton is getting errors when using destructuring and testing via command line? I fetched his branch and the destructuring worked just fine for me.